ERCS/erc-7765.md

```diff
@@ -0,0 +1,170 @@
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+This standard inherits the [ERC-721](./eip-721.md) NFT token standard for all transfer and approval logic. All transfer and approval functions are inherited from this token standard without changes. Additionally, this standard also inherits the ERC-721 Metadata standards for name, symbol, and metadata URI lookup.
+
+Any compliant contract following this EIP **MUST** implement the following interface:
+
+```
+pragma solidity >=0.7.0 <0.9.0;
+
+/// @title ERC-7765 Privileged Non-Fungible Tokens Tied To Real World Assets
+/// @dev See https://eips.ethereum.org/EIPS/eip-7765
+interface IERC7765 /* is IERC721, IERC165 */ {
+
+    /// @notice This event emitted when a specific privilege of a token is successfully exercised.
+    /// @param _operator  the address who exercised the privilege.
+    /// @param _to  the address to benefit from the privilege.
+    /// @param _tokenId  the NFT tokenID.
+    /// @param _privilegeId  the ID of the privileges.
+    event PrivilegeExercised(
+        address indexed _operator,
+        address indexed _to,
+        uint256 indexed _tokenId,
+        uint256 _privilegeId
+    );
+
+    /// @notice This function exercise a specific privilege of a token.
+    /// @dev Throws if `_privilegeId` is not a valid privilegeId.
+    /// @param _to  the address to benefit from the privilege.
+    /// @param _tokenId  the NFT tokenID.
+    /// @param _privilegeId  the ID of the privileges.
+    /// @param _data  extra data passed in for extra message or future extension.
+    function exercisePrivilege(
+        address _to,
+        uint256 _tokenId,
+        uint256 _privilegeId,
+        bytes calldata _data
+    ) external;
+
+    /// @notice This function is to check whether a specific privilege of a token can be exercised.
+    /// @dev Throws if `_privilegeId` is not a valid privilegeId.
+    /// @param _to  the address to benefit from the privilege.
+    /// @param _tokenId  the NFT tokenID.
+    /// @param _privilegeId  the ID of the privileges.
+    function isExercisable(
+        address _to,
+        uint256 _tokenId,
+        uint256 _privilegeId
+    ) external view returns (bool _exercisable);
+
+    /// @notice This function is to check whether a specific privilege of a token has been exercised.
+    /// @dev Throws if `_privilegeId` is not a valid privilegeId.
+    /// @param _to  the address to benefit from the privilege.
+    /// @param _tokenId  the NFT tokenID.
+    /// @param _privilegeId  the ID of the privileges.
+    function isExercised(
+        address _to,
+        uint256 _tokenId,
+        uint256 _privilegeId
+    ) external view returns (bool _exercised);
```

Comment:
Is it possible to exercise a privilege more than once? Because `isExercised` returns only a boolean, I would assume it is "at most once"?

Either way, you should clarify in the document whether privileges can be exercised once or multiple times.